karma.common.conf.js

```diff
@@ -28,16 +24,16 @@ var webpackTestConfig = {
     resolve: webpackCommon.resolve,
     externals: {'jquery': 'jQuery', 'jquery-ui': 'jQuery.ui'},
     module: {
-        loaders: webpackCommon.module.loaders.concat([
+        rules: webpackCommon.module.rules.concat([
             // Assume test files are ES6
             {test: /\.test\.js$/, loader: 'babel-loader'},
         ])
     },
-        node: {
+    node: {
        fs: 'empty'
     }
 };
-
+webpackTestConfig.resolve.descriptionFiles = ["package.json", "bower.json"],
```

Comment:
Drive-by comment: You should be able to use `Object.assign` to merge `webpackCommon.resolve` with more options.

```js
resolve: Object.assign({}, webpackCommon.resolve, {
    descriptionFiles: ['package.json', 'bower.json'],
})
```

Also, as per COSDev, use single-quotes for strings.

Reply:
ah, thanks